commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -75,14 +81,17 @@ def fulfill_order_placed_message_signal_task(
     order_id,
     line_item_state_id,
     source_system,
-    message_id
+    message_id,
+    is_order_fulfillment_redirection_enabled
 ):
+    # pylint: disable=too-many-statements
     """Celery task for fulfilling an order placed message."""
 
     tag = "fulfill_order_placed_message_signal_task"
 
     logger.info(f'[CT-{tag}] Processing order {order_id}, '
-                f'line item {line_item_state_id}, source system {source_system}, message id: {message_id}')
+                f'line item {line_item_state_id}, source system {source_system}, message id: {message_id}, '
+                f'waffle flag: {is_order_fulfillment_redirection_enabled}')
```

Comment:
```suggestion
                f'is_order_fulfillment_forwarding_enabled: {is_order_fulfillment_forwarding_enabled}')
```
we should use the name so that its easier to debug logs.